crmlogin/actualContactEdit.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+Takes in values from contactedit2.php and changes them in the database.
+
+*/
+	include 'Contact.php';
+	$firstname = $_POST['firstname'];
+	$lastname = $_POST['lastname'];
+	$email = $_POST['email'];
+	// $actualEmail = $_POST['actualEmail'];
+	$contactId = $_POST['contactId'];
+	// echo $actualEmail;
+	// echo $firstname;
+	// echo $email;
+	$check = 0;
+
+	if(!empty($_POST['firstname']) || !empty($_POST['lastname']) || !empty($_POST['email'])){
+		$obj = new Contact();
```

Comment:
Here you're instantiating a new object.. but 2 lines later you're using that objects static method.

Static method should be used like this

$contact = Contact::GetById($contactId);